client/app/(auth)/createTeam.tsx

```diff
@@ -0,0 +1,364 @@
+              <Controller
+                control={control}
+                name="name"
+                render={({ field: { onChange, value } }) => (
+                  <View
+                    className={`mb-2 min-h-14 w-full flex-row items-center rounded-xl border ${errors.name ? "border-red-600" : "border-gray-920"} px-2 py-2`}
+                  >
+                    <TextInput
+                      autoCapitalize="none"
+                      className="flex-1 text-base-white placeholder:pbk-b1"
+                      onChangeText={(text) => {
+                        onChange(text);
+                      }}
+                      placeholder="Enter team name"
+                      placeholderTextColor="gray"
+                      value={value}
+                    />
+                    {errors.name && (
+                      <WarningCircle color="#dc2626" size={20} weight="bold" />
+                    )}
+                  </View>
+                )}
+              />
+              {errors.name && (
+                <Text className="pbk-b3 mb-2 text-red-600">
+                  {errors.name.message}
+                </Text>
+              )}
+              <View className="mb-5">
+                <Text className="pbk-b3 text-gray-600">
+                  This is your team name. You can change it later.
+                </Text>
+              </View>
+
+              <Text className="pbk-b2 mb-1.5 text-base-white">
+                Team Abbreviation
+              </Text>
+              <Controller
+                control={control}
+                name="abbreviation"
+                render={({ field: { onChange, value } }) => (
+                  <View
+                    className={`mb-2 min-h-14 w-full flex-row items-center rounded-xl border ${errors.name ? "border-red-600" : "border-gray-920"} px-2 py-2`}
+                  >
+                    <TextInput
+                      autoCapitalize="characters"
+                      className="flex-1 text-base-white placeholder:pbk-b1"
+                      onChangeText={(text) => {
+                        onChange(text);
+                      }}
+                      placeholder="Enter team abbreviation"
+                      placeholderTextColor="gray"
+                      value={value}
+                    />
+                    {errors.abbreviation && (
+                      <WarningCircle color="#dc2626" size={20} weight="bold" />
+                    )}
+                  </View>
+                )}
+              />
+              {errors.abbreviation && (
+                <Text className="pbk-b3 mb-4 text-red-600">
+                  {errors.abbreviation.message}
+                </Text>
+              )}
+
+              <Text className="pbk-b2 mb-1.5 text-base-white">Team logo</Text>
+              <View className="mb-2 mt-4 flex-row items-center justify-between">
+                <Pressable onPress={() => setShowBottomDrawer(true)}>
```

Comment:
I mentioned a bug in the `createProfile` page related to the keyboard and provided a solution that could also fix the issue here. But let me know if you like the explicit call or not